### PR TITLE
doc: add usage section to README.md

### LINKED ISSUE #9: add usage section to README.md
### Enhancement Proposal

The current README.md is basic and does not include any usage examples.  Once things are fleshed out, we should add those.

### PR DESCRIPTION
**Pull request by ca-scribner**
## Issue

Adds simple instructions for using Charmed Istio.  Left out of scope was how to use Ingresses or integrate charms with Beacon, as those should be covered in the respective READMEs (unless we decide to upgrade this README.md to be a global product README.md.  But if that's the case, we should first settle on how to use the other products)

Closes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,17 @@
...
+
+Typically, Istio is deployed to the `istio-system` namespace without any other applications in that namespace.  This is because any Istio Custom Resource deployed to Istio's system namespace are treated as a globally scoped (for example, an `AuthorizationPolicy` deployed in Istio's namespace applies to the entire cluster).  Following these best practices, we can deploy Charmed Istio by:
+
+```bash
+juju add-model istio-system
+juju deploy istio-core-k8s --trust
+```
+
+This deploys the Istio control plane, which can then be used by anything in the Kubernetes cluster.  To get started from here, see:
+* [istio-beacon-k8s](https://github.com/canonical/istio-beacon-operator) for how to quickly integrate charms to the mesh Istio's Ambient mode
```
Comment (mmkay) [inline README.md:23]:
```suggestion
* [istio-beacon-k8s](https://github.com/canonical/istio-beacon-k8s-operator) for how to quickly integrate charms to the mesh Istio's Ambient mode
```

File: README.md [hunk 2]
```diff
@@ -9,3 +9,17 @@
...
+* [istio-ingress-k8s](https://github.com/canonical/istio-beacon-operator) to create ingresses to the cluster
```
Comment (mmkay) [inline README.md:24]:
```suggestion
* [istio-ingress-k8s](https://github.com/canonical/istio-ingress-k8s-operator) to create ingresses to the cluster
```

File: README.md [hunk 3]
```diff
@@ -9,3 +9,17 @@
 [Istio](https://istio.io) is an open source project that implements a service mesh, allowing for a way to observe and control the traffic flow between applications in Kubernetes.  Istio is a key tool in securing Kubernetess workloads and hardening your environment.
 
 The istio-core Charmed Operator deploys and manages the Istio control plane components in a Kubernetes cluster.  The operator is designed to be used in conjunction with the [istio-beacon-k8s](https://github.com/canonical/istio-beacon-operator) and [istio-ingress-k8s](https://github.com/canonical/istio-beacon-operator) charms to deploy and configure Istio using Juju.
```
Comment (mmkay) [inline README.md:11]:
```suggestion
The istio-core Charmed Operator deploys and manages the Istio control plane components in a Kubernetes cluster.  The operator is designed to be used in conjunction with the [istio-beacon-k8s](https://github.com/canonical/istio-beacon-k8s-operator) and [istio-ingress-k8s](https://github.com/canonical/istio-ingress-k8s-operator) charms to deploy and configure Istio using Juju.
```

**Review by mmkay — COMMENTED** [review]
A couple link fixes would be needed.

**Review by dstathis — APPROVED** [review]
Good to go once the links are fixed.

**Pull request merged**